references.bib

```diff
@@ -0,0 +1,168 @@
+
+@misc{monolith2017shadow,
+  title={Middle-earth: Shadow of War},
+  publisher={Waner Bros. Interactive Entertainment},
+  organization={Monolith Productions},
+  year={2017}
+}
+
+
+@misc{ubisoft2013assassins,
+  title={Assassin's Creed IV: Black Flag},
```

Comment:
You need to wrap this in an extra pair of curly brackets to prevent BibTeX from messing with the capitalisation, i.e.
```
title={{Assassin's Creed IV: Black Flag}}
```